dev_guide/compute_resources.adoc

```diff
@@ -279,3 +279,13 @@ To specify compute resources via the CLI:
 $ oc run nginx --image=nginx --limits=cpu=200m,memory=400Mi --requests=cpu=100m,memory=200Mi
 ----
 ====
+
+ifdef::openshift-enterprise,openshift-origin[]
```

Comment:
@sosiouxme You can add `openshift-dedicated` here, we have a Cluster Administrator guide for Dedicated now:

https://docs.openshift.com/dedicated/latest/admin_guide/index.html